src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -163,7 +163,7 @@
 
     private transient KeyPair usableKeyPair;
 
-    protected transient AmazonEC2 connection;
+    protected volatile transient AmazonEC2 connection;
```

Comment:
```suggestion
    protected transient volatile AmazonEC2 connection;
```

More common word ordering in Java.